remote-query/pom.xml

```diff
@@ -39,7 +39,7 @@
 
         <!-- Defines the main class to be execute when running java -jar command -->
         <main.class.remote-query>org.jboss.as.quickstarts.datagrid.hotrod.query.AddressBookManager</main.class.remote-query>
-        <version.org.infinispan>7.0.0.Final</version.org.infinispan>
+        <version.org.infinispan>7.1.0-SNAPSHOT</version.org.infinispan>
```

Comment:
Those Quickstarts need to be built using some existing Infinispan version, which is available in our repository. Could you please change it?